src/modules/endpoint/sagas/loadEvents.js

```diff
@@ -23,19 +23,25 @@ function * fetchEvents (city: string, language: string): Saga<?Array<EventModel>
 
 function * loadEvents (
   city: string, language: string, dataContainer: DataContainer, shouldUpdate: boolean): Saga<FetchMapType> {
-  if (dataContainer.eventsAvailable() && !shouldUpdate) {
-    console.debug('Using cached events')
-    return {}
-  }
+  let events: ?Array<EventModel>
+
+  // If data was loaded and should not be updated, return
```

Comment:
Where's this return that you announce here? :D

Reply:
Seems like a Copy&Pasta mistake :D